TradeItIosTicketSDK2/TradeItNumericInputCell.swift

```diff
@@ -2,31 +2,56 @@ import UIKit
 
 class TradeItNumericInputCell: UITableViewCell {
     @IBOutlet weak var textField: TradeItNumberField!
-
+    
     var onValueUpdated: ((_ newValue: NSDecimalNumber?) -> Void)?
-
+    
     override func awakeFromNib() {
         self.textField.padding = UIEdgeInsets(top: 0, left: 5, bottom: 0, right: 5)
+        self.addDoneButtonOnKeyboard()
     }
-
+    
+    
+    
     func configure(initialValue: NSDecimalNumber?,
                    placeholderText: String,
                    onValueUpdated: @escaping (_ newValue: NSDecimalNumber?) -> Void) {
         self.onValueUpdated = onValueUpdated
         self.textField.placeholder = placeholderText
-
+        
         if let initialValue = initialValue {
             self.textField.text = "\(initialValue)"
         } else {
             self.textField.text = nil
         }
     }
-
+    
+    func addDoneButtonOnKeyboard(){
+        let doneToolbar: UIToolbar = UIToolbar(frame: CGRect(x: 0, y: 0, width: 320, height: 50))
```

Comment:
<img width="355" alt="screen shot 2017-06-15 at 3 26 20 pm" src="https://user-images.githubusercontent.com/22267174/27198152-06cc0cc0-51df-11e7-902e-e90bd2c99c60.png">


I tested it on an iPhone 5, and it seems to look relatively fine